glsl/src/parse_tests.rs

```diff
@@ -2322,104 +3324,137 @@ fn parse_layout_buffer_block_0() {
       syntax::TypeQualifierSpec::Storage(syntax::StorageQualifier::Buffer),
     ]),
   };
-  let block = syntax::ExternalDeclaration::Declaration(syntax::Declaration::Block(syntax::Block {
-    qualifier: type_qual,
-    name: "Foo".into(),
-    fields: vec![syntax::StructFieldSpecifier {
-      qualifier: None,
-      ty: syntax::TypeSpecifier {
-        ty: syntax::TypeSpecifierNonArray::TypeName("char".into()),
-        array_specifier: None,
-      },
-      identifiers: syntax::NonEmpty(vec!["a".into()]),
-    }],
-    identifier: Some("foo".into()),
-  }));
+  let block = syntax::Node {
+    contents: syntax::ExternalDeclaration::Declaration(syntax::Declaration::Block(syntax::Block {
+      qualifier: type_qual,
+      name: "Foo".into(),
+      fields: vec![syntax::StructFieldSpecifier {
+        qualifier: None,
+        ty: syntax::TypeSpecifier {
+          ty: syntax::TypeSpecifierNonArray::TypeName("char".into()),
+          array_specifier: None,
+        },
+        identifiers: syntax::NonEmpty(vec!["a".into()]),
+      }],
+      identifier: Some("foo".into()),
+    })),
+    span: syntax::NodeSpan {
+      offset: 0,
+      line: 1,
+      length: src.len() - 1, // Final newline
+    },
+  };
 
   let expected = syntax::TranslationUnit(syntax::NonEmpty(vec![block]));
 
-  assert_eq!(translation_unit(src), Ok(("", expected)));
+  assert_eq_parser(translation_unit, src, Ok(("", expected)), &ctx);
```

Comment:
Currently broken on Windows because of how new lines are checked out by git with autocrlf on